MekHQ/src/mekhq/campaign/personnel/generator/DefaultSkillGenerator.java

```diff
@@ -55,7 +55,8 @@ public void generateSkills(Person person, int expLvl) {
             generateDefaultSkills(person, secondaryRole, expLvl, bonus, mod);
         }
 
-        bonus = getPhenotypeBonus(person);
+        // apply phenotype bonus only to primary skills
+        bonus = 0;
 
```

Comment:
This feels like it would profit from getting rid of the bonus variable altogether and using the method and 0 directly as appropriate.

Reply:
It's a non-trivial method, so storing in a variable makes sense given it is used in two separate methods previously. I also think using the variable bonus is more readable than a bunch of inline magic numbers when it isn't related to graphical code.